projects/storefrontapp-e2e-cypress/cypress/e2e/regression/coupons/my-coupons.e2e-spec-flaky.cy.ts

```diff
@@ -34,6 +34,16 @@ viewportContext(['mobile', 'desktop'], () => {
     });
   });
 
+  describe('My coupons - claim coupons with code in body using authenticated user', () => {
```

Comment:
I think this tests will break our pipeline, before the CI server will not be updated to the cx version that has new endpoint. Could you please skip it for now and add a comment that should be uskiped when CI server will be updated to **** version. (I'm not sure which version it will be, you knows better)

Reply:
Hi @rmch91 , thanks for remind, think the CI server already included the new occ endpoints because we integrated the new endpoints into Commerce since 2211.28, see [PR](https://github.tools.sap/cx-commerce/promotionengine/pull/1071/files#diff-2e293e5f650a5b97df65928614b8952bacd6d1a7c7781a101dbbeb4bf5935c28).
Also I locally run e-2-e cypress test which based on default CX_BASE_URL, result like followings:
<img width="720" alt="Screenshot 2024-12-19 at 13 44 30" src="https://github.com/user-attachments/assets/f7ef5614-f3e5-496b-9dcf-222b1df07142" />
but has following 2 failed cases, think it more related with env settings(still have these failed cases based on develop branch), instead of caused by my changes.
<img width="727" alt="Screenshot 2024-12-19 at 13 58 07" src="https://github.com/user-attachments/assets/33443f7c-4b2b-468f-b60d-139f2dcd78d3" />
![image](https://github.com/user-attachments/assets/d2c86418-b8d7-44ed-82f0-1379a26b8b06)

Maybe the way I run e2e test is not right?(also you can tell me right way) or  you can try run these e2e test cases(based on fix branch) to check whether it breaking case in CI. or other ideas from your side?
Thanks a lot.

Reply:
thanks for confirmation.